packages/hash/frontend/src/pages/[account-slug]/types/entity-type/edit-bar.tsx

```diff
@@ -1,10 +1,11 @@
 import { faSmile } from "@fortawesome/free-regular-svg-icons";
 import { FontAwesomeIcon } from "@hashintel/hash-design-system";
 import { Box, Collapse, Container, Stack, Typography } from "@mui/material";
-import { ReactNode, useEffect, useRef, useState } from "react";
+import { ReactNode, useContext, useEffect, useRef, useState } from "react";
 import { useFormState } from "react-hook-form";
 import { PencilSimpleLine } from "../../../../shared/icons/svg";
 import { Button, ButtonProps } from "../../../../shared/ui/button";
+import { EditBarContext } from "../../../../shared/layout/layout-with-sidebar/edit-bar-scroller";
```

Comment:
N.B.: should be `import { EditBarContext } from "../../../../shared/×××`"` (no need to fix this now)